package.json

```diff
@@ -14,7 +14,8 @@
     "ci": "npm-run-all --parallel lint coveralls"
   },
   "peerDependencies": {
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
```

Comment:
Do you still need lodash ?
if not remove it from here and rollup config

Reply:
helpers `get` and `template` needed

Reply:
ho yes I'm stupid :x